pkg/remoteconfig/state/products/apmsampling/remote_rates.go

```diff
@@ -27,7 +27,16 @@ type TargetTPS struct {
 	Mechanism SamplingMechanism `msgpack:"4"`
 }
 
+type RareSamplerConfig string
```

Comment:
There may be a better way of building a type with the following requirements:
 - default "unset" value different from the "disabled" value
 - less error prone than incrementing ints (but maybe I'm overthinking)